jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6256,47 +6259,172 @@
+						},
+						"description": "Creating a new JVM for JDK upgrade testing."
+					},
+					"response": []
+				},
+				
+				{
+					"name": "upgrade JDK- Negative Test case",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 500\"] = responseCode.code === 500;",
+											"",
+									"var jsonData = JSON.parse(responseBody);",
+									"tests[\"Assert 500 error\"] = jsonData.status === 500;",
+
+									"tests[\"Assert No setenv.bat error\"] = jsonData.message === \"The remote JVM ::Postman-test-jvm-jdk-upgrade must be in STOPPED state before attempting to upgrade the JDK of the JVM\";"
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/jvms/{{JDKUpgradeJvmName}}/jdk",
+						"method": "PUT",
 						"header": [],
 						"body": {
-							"mode": "formdata",
-							"formdata": [
-								{
-									"key": "metaData",
-									"type": "file",
-									"src": "../resources/postman/setenv_upgradeJDK.bat.json",
-									"value": ""
-								},
-								{
-									"key": "tplFile",
-									"type": "file",
-									"src": "../resources/postman/setenv.bat.tpl",
-									"value": ""
-								}
-							]
+							"mode": "raw",
+							"raw": ""
 						},
-						"description": ""
+						"description": "This webservices tries to upgrade the JDK version of the JVM which is not yet generated."
 					},
 					"response": []
 				},
 				{
-					"name": "create Tomcat media for JVM-upgrade",
+					"name": "generate jvm",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 200\"] = responseCode.code === 200;"
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/jvms/{{JDKUpgradeJvmName}}/conf",
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"description": "Generating the JDK upgrade JVM. "
+					},
+					"response": []
+				},
+				{
+					"name": "upgrade JDK- Negative Test case",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 500\"] = responseCode.code === 500;",
+											"",
+									"var jsonData = JSON.parse(responseBody);",
+									"tests[\"Assert 500 error\"] = jsonData.status === 500;",
+
+									"tests[\"Assert No setenv.bat error\"] = jsonData.message === \"setenv.bat resource file does not exist for JVM: Postman-test-jvm-jdk-upgrade ,and the file is required to set the JAVA_HOME for the JVM.\";"
```

Comment:
Again, don't hard code the JVM name here.

Reply:
Also change the formatting of the message:
"The setenv.bat file does not exist as a resource for JVM: Postman-test-jvm-jdk-upgrade. Please create a setenv.bat resource for the JVM in order to set the JAVA_HOME environment variable."

Reply:
Will this fail for Linux installs since the JVMs will be configured with a setenv.sh?